### PR TITLE
Adds UI for admin sales : fees route

### LINKED ISSUE #244: Add UI for admin/sales/fees
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

screenshot from 2017-06-17 10-31-33

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

Adds UI for `admin/sales/fees`
#### Changes proposed in this pull request:

image

Fixes #244

### CODE REVIEW

**Comment by CosmicCoder96** [comment]
@geekyd Please review

**Comment by geekyd** [comment]
@CosmicCoder96 Please provide deployment link. Thanks

**Comment by CosmicCoder96** [comment]
@geekyd https://cc96-opev-frontend.herokuapp.com/admin/sales/fees

**Comment by CosmicCoder96** [comment]
@niranjan94 Please review

File: app/templates/admin/sales/fees.hbs [hunk 1]
```diff
@@ -1 +1,67 @@
-{{outlet}}
+<div class="ui stackable grid">
+  <div class="row">
+    <h1 class="ui header six wide column">
```
Comment (geekyd) [inline app/templates/admin/sales/fees.hbs]: Please use `h2` tag here & replace `six wide` class.

Comment (CosmicCoder96) [inline app/templates/admin/sales/fees.hbs]: done

**Comment by CosmicCoder96** [comment]
@geekyd Please review, have made the changes

**Comment by geekyd** [comment]
@CosmicCoder96 Please update the branch & test for linting issue as we have added new version of ember-cli-eslint.

**Comment by CosmicCoder96** [comment]
@geekyd Done, it's passing the build

**Pull request merged**